src/Hintman/Suggestion/TrailingSpaces.hs

```diff
@@ -0,0 +1,13 @@
+module Hintman.Suggestion.TrailingSpaces
+       ( suggest
+       ) where
+
+import Data.Text (stripEnd)
+import Hintman.Suggestion.Core (Line (..), Suggest (..))
+
+
+suggest :: FilePath -> [Line] -> [Suggest]
+suggest path =
+    foldr (\Line{..} acc ->
+        if lineBody == stripEnd lineBody then acc
```

Comment:
You should put `stripEnd lineBody` into var, so you won't do this action twice (here and the next line)